### ISSUE TITLE
Error when gathering files

### ISSUE THREAD

**Issue by shomilsaini** [jbrodriguez/unbalance#97]
When gathering files I get this error.
Scenario:
1. Open Unbalance & Select Gather Page
2. Select the folder to gather and press next
3. On Plan monitor page press next
4. Click to select Target disk
5. Error (see below) appears with URL : http://xxx.xxx.xxx.xxx:7090/gather/transfer/targets

Selected media was only on 1 drive. Target media is same drive, effectively 0b transfer required.

Expected: it should either not allow selecting same drive where no transfer is required or when click next after target selected should show complete page instantly.

```
Unexpected Application Error!
Cannot read properties of null (reading 'map')
TypeError: Cannot read properties of null (reading 'map')
    at I1 (http://xxx.xxx.xxx.xxx:7090/assets/index-d19XgG5A.js:124:81626)
    at tf (http://xxx.xxx.xxx.xxx:7090/assets/index-d19XgG5A.js:38:19519)
    at zu (http://xxx.xxx.xxx.xxx:7090/assets/index-d19XgG5A.js:40:3139)
    at Hg (http://xxx.xxx.xxx.xxx:7090/assets/index-d19XgG5A.js:40:44737)
    at Fg (http://xxx.xxx.xxx.xxx:7090/assets/index-d19XgG5A.js:40:39727)
    at v2 (http://xxx.xxx.xxx.xxx:7090/assets/index-d19XgG5A.js:40:39655)
    at tl (http://xxx.xxx.xxx.xxx:7090/assets/index-d19XgG5A.js:40:39508)
    at Yu (http://xxx.xxx.xxx.xxx:7090/assets/index-d19XgG5A.js:40:35875)
    at gh (http://xxx.xxx.xxx.xxx:7090/assets/index-d19XgG5A.js:40:36678)
    at hr (http://xxx.xxx.xxx.xxx:7090/assets/index-d19XgG5A.js:38:3274)
```

**Comment by jbrodriguez** [comment]
hey ! so it allowed you to select the same target as the origin ? that sounds a bit weird

can you share a screenshot right before you click target disk (mask any private info as you need) ?

**Comment by shomilsaini** [comment]
As soon as you click on disk the error appears. 
Same disk as source. 
Thanks!

**Comment by jbrodriguez** [comment]
thanks for bringing this up !

fixed with #99 

will publish soon

**Comment by shomilsaini** [comment]
> thanks for bringing this up !
> 
> fixed with #99
> 
> will publish soon

Thank you! :)
